### PR TITLE
update licence and lint - add some refs for consumers

### PR DESCRIPTION
**Pull request by daraghmartin**

### CODE REVIEW

File: packages/@cosmos-building-blocks/app-ec2/package.json [hunk 1]
```diff
@@ -43,19 +43,19 @@
     "source-map-support": "^0.5.16"
   },
   "dependencies": {
```
Comment (timpur) [inline packages/@cosmos-building-blocks/app-ec2/package.json]: We need to fix this as peer, other wise it wont work with the rest of the lib, also version updating should happen across the libs not just for one lib.

File: packages/@cosmos-building-blocks/app-ec2/.eslintrc [hunk 2]
```diff
@@ -18,8 +18,10 @@
         "imports": "always-multiline",
```
Comment (timpur) [inline packages/@cosmos-building-blocks/app-ec2/.eslintrc]: The think we need to move lint out to root, as their should only be one "standard" ?

File: packages/@cosmos-building-blocks/app-ec2/package.json (hunk 1 above)
Comment (daraghmartin) [reply packages/@cosmos-building-blocks/app-ec2/package.json]: no actual peering yet - lets chat and update later

File: packages/@cosmos-building-blocks/app-ec2/.eslintrc (hunk 2 above)
Comment (daraghmartin) [reply packages/@cosmos-building-blocks/app-ec2/.eslintrc]: not sure I want the overrides to be global.  but good discussion to have

**Pull request merged**